salt/tests/unit/modules/files/test_metalk8s_kubernetes.yaml

```diff
@@ -11,7 +11,7 @@ common_tests:
       a: b
     info_scope: null
     raises: True
-    result: "Invalid manifest: An error has occurred"
+    result: "Invalid manifest"
```

Comment:
In tests, could be useful to check that we can find the original error in the traceback. Maybe to tackle later, as debt.

Reply:
Hmm you right, I will try to do something about it

Reply:
Ok I think we will do it later I will open a ticket for this, basically you need to check the content of `__cause__` in the exception so will likely need to create a new `assertRaises...` to handle it (#3086)

Reply:
Perfect, thanks!